autoinstall/FreeBSD/installerconfig

```diff
@@ -0,0 +1,140 @@
+
+debug=yes
+nonInteractive=yes
+
+{% if boot_disk_controller == 'nvme' %}
+PARTITIONS="nvd0 gpt"
+{% elif boot_disk_controller == 'sata' or boot_disk_controller == 'ide' %}
+PARTITIONS="ada0 gpt"
+{% else %}
+PARTITIONS="da0 gpt"
+{% endif %}
+
+{% if guest_id.find('64Guest') != -1 %}
+DISTRIBUTIONS="kernel.txz base.txz kernel-dbg.txz lib32.txz src.txz ports.txz"
+{% else %}
+DISTRIBUTIONS="kernel.txz base.txz kernel-dbg.txz src.txz ports.txz"
+{% endif %}
+
+#!/bin/sh
+# Set hostname
+sysrc hostname="FreeBSD-{{ hostname_timestamp }}"
+
+{% if boot_disk_controller == 'nvme' %}
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 nvd0
+{% elif boot_disk_controller == 'sata' or boot_disk_controller == 'ide' %}
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 ada0
+{% else %}
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 da0
+{% endif %}
+
+# Set Time Zone to UTC
+echo "Setting Time Zone to UTC ..."
+/bin/cp /usr/share/zoneinfo/UTC /etc/localtime
+/usr/bin/touch /etc/wall_cmos_clock
+/sbin/adjkerntz -a
+
+echo "Set network interface with DHCP IP assignment ..." > /dev/ttyu0
+ifdev=$(ifconfig | grep '^[a-z]' | cut -d: -f1 | head -n 1)
+echo "Get ifname ${ifdev}" > /dev/ttyu0
+sysrc ifconfig_${ifdev}=DHCP
+
+# Get DHCP for nic0
+echo "Get IP with dhclient ..." > /dev/ttyu0
+dhclient ${ifdev}
+sleep 15
+echo "Check network ..." > /dev/ttyu0
+ifconfig > /dev/ttyu0
+
+# Set Proxy. without the proxy, maybe it will take more time to download pakages xorg & gnome
+{% if http_proxy_vm is defined and http_proxy_vm %}
+setenv HTTP_PROXY {{ http_proxy_vm }}
+{% endif %}
+
+# Installing packages
+echo "Installing packages ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -y
+
+# Hit issue: reset by peer during install packages
+# The open-vm-tools is not installed by default
+mkdir -p /usr/local/etc/pkg/repos
+mount > /dev/ttyu0
+cp -rf /dist/packages/repos/FreeBSD_install_cdrom.conf /usr/local/etc/pkg/repos/FreeBSD_install_cdrom.conf
+env ASSUME_ALWAYS_YES=YES pkg update -f > /dev/ttyu0
+
+# We install packages from ISO image
+# Different packages between the 32bit image and 64bit image
+packages_to_install='bash sudo xorg kde5'
+for package_to_install in $packages_to_install
+do
+    echo "Install package $package_to_install (try $try_count time) ..." > /dev/ttyu0
+    env ASSUME_ALWAYS_YES=YES pkg install -y $package_to_install
+    ret=$?
+    if [ $ret == 0 ]
+    then    
+        echo "Succeed to install the package($package_to_install)" > /dev/ttyu0
+    else
+        echo "Failed to install the package($package_to_install)" > /dev/ttyu0
+    fi
+done
+
+# Disable ISO repo and enable default repo
+rm -rf /usr/local/etc/pkg/repos/FreeBSD_install_cdrom.conf
+env ASSUME_ALWAYS_YES=YES pkg update -f > /dev/ttyu0
+packages_to_install='sddm open-vm-tools xf86-video-vmware xf86-input-vmmouse wget curl'
```

Comment:
I still suggest you installing xf86-video-vmware from local repo so that we can collect the builtin driver version in check_inbox_driver test case in future.

Reply:
Done. Thanks